src/reverse-numbers.test.js

```diff
@@ -0,0 +1,11 @@
+const reverse = require('./reverse-numbers');
```

Comment:
delele this due to miss understand requirement

Reply:
okay, I did.